evap/static/ts/src/notebook.ts

```diff
@@ -50,28 +45,36 @@ export class NotebookLogic {
     private readonly evapContent: HTMLElement;
     private readonly formLogic: NotebookFormLogic;
     private readonly localStorageKey: string;
-
-    constructor(notebookSelector: string) {
-        this.notebookCard = selectOrError(notebookSelector);
-        this.formLogic = new NotebookFormLogic(NOTEBOOK_FORM_SELECTOR);
-        this.evapContent = selectOrError(WEBSITE_CONTENT_SELECTOR);
-        this.localStorageKey = NOTEBOOK_LOCALSTORAGE_KEY + "_" + notebookSelector;
+    private readonly collapseNotebookButton: HTMLElement;
+
+    constructor(
+        notebook: HTMLElement,
+        noteBookForm: HTMLFormElement,
```

Comment:
inconsistent capitalization compared to all other instances of "notebook" in names